tests/test_fields.py

```diff
@@ -237,3 +237,17 @@ def test_update_one_attribute(self):
             hmac_field__hash_of=expected,
         )
         self.assertEqual(updated_instance.first(), instance)
+
+    def test_pgp_public_key_negative_number(self):
+        """Assert negative value is saved with an `IntegerPGPPublicKeyField` field."""
+        expected = -1
+        instance = EncryptedTextFieldModelFactory.create(integer_pgp_pub_field=expected)
```

Comment:
This is what allowed to have the `ProgrammingError` as it was happening when saving a model with an integer field.